test/org/sagebionetworks/bridge/play/controllers/ConsentControllerMockedTest.java

```diff
@@ -19,36 +26,84 @@
 import org.sagebionetworks.bridge.services.ParticipantOptionsService;
 import org.sagebionetworks.bridge.services.StudyService;
 
+import com.fasterxml.jackson.databind.JsonNode;
+
+import play.mvc.Result;
+import play.test.Helpers;
+
 public class ConsentControllerMockedTest {
```

Comment:
Is it worth adding a unit test to verify that we successfully add the signedOn date?